### PR TITLE
make CRI test logs persistent

### PR DESCRIPTION
**Pull request by shyamjesal**
- [x] CRI test
- [ ] Minio test
- [ ] Stock containerd test

```
Run echo $HOSTNAME
cri-test-github-runner-iccluster128-1-control-plane
691114828
```

The logs are stored on iccluster128 at /tmp/ctrd-logs/691114828/

Signed-off-by: Shyam Jesal <s.jesalpura@gmail.com>

### CODE REVIEW

File: cri/Makefile [hunk 1]
```diff
@@ -21,14 +21,14 @@
...
-CTRDLOGDIR:=/tmp/ctrd-logs
+CTRDLOGDIR:=/tmp/ctrd-logs/${GITHUB_RUN_ID}
 test:
 	sudo mkdir -p -m777 -p $(CTRDLOGDIR)
 	sudo containerd 1>$(CTRDLOGDIR)/ctrd.out 2>$(CTRDLOGDIR)/ctrd.err &
 	sleep 1s
 	sudo /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 1>$(CTRDLOGDIR)/fccd.out 2>$(CTRDLOGDIR)/fccd.err &
 	sleep 1s
-	sudo ./../vhive 1>$(CTRDLOGDIR)/orch.out 2>$(CTRDLOGDIR)/orch.err &
+	sudo ./../vhive -d 1>$(CTRDLOGDIR)/orch.out 2>$(CTRDLOGDIR)/orch.err &
```
Comment (ustiugov) [inline cri/Makefile:31]: must be -dbg if I remember correctly

File: cri/Makefile [hunk 2]
```diff
@@ -21,14 +21,14 @@
 # SOFTWARE.
 
 EXTRAGOARGS:=-v -race -cover
-CTRDLOGDIR:=/tmp/ctrd-logs
+CTRDLOGDIR:=/tmp/ctrd-logs/${GITHUB_RUN_ID}
```
Comment (ustiugov) [inline cri/Makefile:24]: is this folder available on the host and where? please leave a comment

Comment (ustiugov) [reply cri/Makefile:24]: why not ${{ github.run_id }}-${{ github.run_number }} here as well?

Comment (shyamjesal) [reply cri/Makefile:24]: ${{ github.run_id }}-${{ github.run_number }} is stored in an evironment variable [here](https://github.com/ease-lab/vhive/blob/2b9439af058693532f55e6ecd5a28aca6c205ca1/.github/workflows/cri_test.yml#L55) which is then passed on to the makefile.

**Review by ustiugov — DISMISSED** [review]

**Review by ustiugov — APPROVED** [review]

**Pull request merged**